Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -332,13 +332,9 @@ internal static async Task RunCoreTests(MinioClient minioClient)
             GetBucketPolicy_Test1(minioClient)
         };
 
-        ParallelOptions parallelOptions = new()
-        {
-            MaxDegreeOfParallelism = 4
-        };
-        await Parallel
-            .ForEachAsync(coreTestsTasks, parallelOptions,
-                async (task, _) => { await task.ConfigureAwait(false); }).ConfigureAwait(false);
+        var MaxNoOfParallelProcesses = 4;
```

Comment:
No need to put it in a local var, just add it in the util func and remove this from the method